src/Components/WebView/WebView/src/FileExtensionContentTypeProvider.cs

```diff
@@ -152,7 +152,7 @@ public FileExtensionContentTypeProvider()
                 { ".jpe", "image/jpeg" },
                 { ".jpeg", "image/jpeg" },
                 { ".jpg", "image/jpeg" },
-                { ".js", "application/javascript" },
+                { ".js", "text/javascript" },
```

Comment:
@dotnet/dotnet-maui-blazor-eng - we need to verify this works in all the WebViews. I'd be very surprised if it _doesn't_, but we need to positively verify that it does.

Reply:
In particular, things like older Androids with older Chromes, which we do still support.

Reply:
@Eilon Makes sense. I'm not familiar with your test infrastructure for that stuff; would someone on your team be able to do that test pass before this PR is merged, or would this be picked up post-merge? I'm happy to hold off on merging this if needed, just let me know how to proceed :).

Reply:
Pinging @TanayParikh @javiercn @MackinnonBuck who can help you with this.